### ISSUE TITLE
Nunchuk getting stuck upwards

### ISSUE THREAD

**Issue by lawik** [xerpi/nunchuk2pi#1]
I got it connected and upon restarting the driver it usually works for second, all directions. Then it acts like the nunchuk stick is pushed fully forward and the cursor gets stuck along the top of the screen. It could be hardware but was curious if you've seen this before.

**Comment by xerpi** [comment]
I think the latest versions of Linux feature the nunchuk driver in the kernel itself, so maybe it would be a better idea to check that out since I did this project a long time ago.

**Comment by lawik** [comment]
Fair enough, tried a second one now that also had the same behavior. Will check for that driver instead.

**Comment by lawik** [comment]
On looking around some I can't seem to find any particular reference to a driver that is available for rpi..
